javascript/javascript1/week2/student.js

```diff
@@ -0,0 +1,32 @@
+const class07Students = [];
+
+function addStudentToClass(studentName) {
+  if (class07Students.length === 6 && studentName !== "Queen Margrethe II") {
+    console.log("Cannot add more students to class 07");
+    return;
+  } else if (class07Students.includes(studentName)) {
+    console.log(`Student ${studentName} is already in the class`);
+    return;
+  } else if (!studentName) {
```

Comment:
Just noticed that when you return inside the `if` condition you don't need to do the `else`, like so:
```js
function addStudentToClass(studentName) {
  if (class07Students.length === 6 && studentName !== "Queen Margrethe II") {
    console.log("Cannot add more students to class 07");
    return;
  }

  if (class07Students.includes(studentName)) {
    ...
```